src/main/groovy/org/shipkit/gradle/notes/UpdateReleaseNotesTask.java

```diff
@@ -129,6 +129,13 @@ public String getGitHubRepository() {
     }
 
     /**
+     * @return a link to the generated release notes file hosted on Github.
+     */
+    public String getReleaseNotesUrl(String branch) {
```

Comment:
I'm wondering if this is unit testable easily. If so, let's a add a test. String concat is a complexity worth unit testing.

Reply:
yes, indeed, we should add a test. I will do so.